freetar/templates/index.html

```diff
@@ -71,9 +71,16 @@
     <button type="button" class="btn btn-primary autoscroll" onclick="scroll_timeout+=50" >-</button>
     <i id="dark_mode" class="fa fa-adjust fa-fw"></i>
     <i class="fa fa--regular fa-star favorite"></i>
+
+    <div id="transpose">
+        <span>Transpose</span>
```

Comment:
The transposition works quiet well. Can you update the chord visulization after transposition?

Reply:
Also I think we need to restructure the menu and use more of the space on the right side. But we can do this later

Reply:
Updating the chord visualisation after transposition would require either having a specific endpoint to get the chord diagrams or rendering them in JS rather than getting them from SongDetail